app/services/service_listeners/publishing_api_pusher.rb

```diff
@@ -39,6 +44,24 @@ def handle_html_attachments(event)
       PublishingApiHtmlAttachmentsWorker.perform_async(edition.id, event)
     end
 
+    # If the previous edition had extra translations, redirect them to the :en locale.
+    # Unmigrated formats as of now (02/11/2016) do not work with this, as the redirects
+    # are not removed if the translation is added back in.
+    def handle_translations
+      is_migrated_format = edition.rendering_app != Whitehall::RenderingApp::WHITEHALL_FRONTEND
+      previous_edition = edition.previous_edition
```

Comment:
We shouldn't load the `.previous_edition` if we already know `is_migrated_format == false`. This would be better inside `if is_migrated_format`